### ISSUE TITLE
[Feature Request] Show read write speeds for multiple disks.

### ISSUE THREAD

**Issue by LampPrinter** [exelban/stats#698]
I use stats to show read/write transfer speeds. But right now you can only select 1 disk to show stats (I have selected Internal). I always have an external drive connected. Features I would like -
1) Combined disk speeds in status bar for multiple disks.
2) Individual disk speed in dropdown.
3) A threshold, maybe do not show speeds less than 1-2Mbps in status bar.

Please consider implementing them :)

**Comment by exelban** [comment]
Hi. Thanks for the feedback.
The first two features are already described in the issues. The threshold for speed? Why did it need? I don't see any reason why it must be implemented.

**Comment by LampPrinter** [comment]
It is fine. The first 2 ones are the important ones ;)

**Comment by exelban** [comment]
TLDR: first cannot be implemented, the second could be. But I need a design for it.

**Comment by LampPrinter** [comment]
This is how iStats does it.

Per Disk

<img width="346" alt="Detailed Per Disk" src="https://user-images.githubusercontent.com/7559328/141692276-3ae07dea-5821-4c2e-adfe-b40eac21bcb8.png">

Combined

<img width="346" alt="Detailed Combined" src="https://user-images.githubusercontent.com/7559328/141692291-a1334007-07c4-4778-b76c-441efe854122.png">
